doc/list.rst

```diff
@@ -2339,6 +2339,14 @@ List of Available Rules
 
    *warning risky* Fixer could be risky if one is calling data provider function dynamically.
 
+   Configuration options:
+
+   - | ``force``
+     | whether to make static data providers having dynamic class calls
```

Comment:
please add a description that it can do change that will break the code and one will need to manually do the adjustment of method to make it work in static context, and what's going on here.

[i like the idea of this option, but we need good description to let ppl know when to use it]

Reply:
suggestion as a started, but we can have then the part "this will break your code and you need to manually adjust code to..."
```suggest
     | whether to make static the data providers even when they are having dynamic class calls [warning: this will break... you need to adjust code manually by ...]
```